rhc-ose-ansible/roles/disconnected-git/tasks/main.yaml

```diff
@@ -0,0 +1,65 @@
+---
+- name: Install Packages
+  action: "{{ ansible_pkg_mgr }} name={{ item }} state=present"
+  with_items:
+    - git
+    - httpd
+    - firewalld
+    
```

Comment:
need to add `libsemanage-python` to this list of packages to install as its a dependancy for the httpd config task.